feature-libs/organization/administration/assets/translations/en/units.i18n.ts

```diff
@@ -167,7 +167,7 @@ export const orgUnitUsers = {
 };
 
 export const orgUnitUserRoles = {
-  title: 'User roles',
+  title: 'User roles and rights',
```

Comment:
For a constant named `orgUnitUserRoles` I'd expect its title to be `User roles` as it is and not `User roles and rights`, perhaps we can create another constant named `orgUnitUserRights` or so? Or maybe create another constant `orgUnitUserRolesAndRights`?